Gopkg.toml

```diff
@@ -11,3 +11,7 @@ required = [
 [[constraint]]
   name = "github.com/percona/pmm"
   branch = "PMM-2.0"
+
+[[constraint]]
+name = "github.com/percona-platform/saas"
+branch = "PMM-7279-param-units"
```

Comment:
TODO

Reply:
the saas PR was merged please update this

Reply:
Done